src/SystemCommands-RefactoringSupport/TRefactoringCommandSupport.trait.st

```diff
@@ -17,5 +17,11 @@ TRefactoringCommandSupport >> setUpModelFromContext: aContext [
 	| browser scope |
 	browser := aContext browser.
 	scope := browser defaultNavigationScope.
+	scope description = 'current image' ifTrue: [ 
```

Comment:
```suggestion
	scope isCurrentImage ifTrue: [ 
```
Even if this method not exist it is how you should approach such cases. Code should not rely on string comparisons.

Reply:
I understand, thank you